LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015-2018 Kong Inc. (https://konghq.com)
+Copyright © 2015-2018 Kong Inc. (https://konghq.com)
```

Comment:
Should this be updated to year 2020? @hbagdi

Reply:
Indeed. And `(c)` is good as is.

Reply:
Thank you @hbagdi !